Dockerfile

```diff
@@ -1,39 +1,58 @@
-FROM alpine:latest
+FROM centos:latest
 LABEL maintainer="OSC"
 
 # Set language to avoid bugs that sometimes appear
 ENV LANG en_US.UTF-8
 
 # Set up requirements
-RUN apk upgrade --update \
-    && apk --no-cache add \
-          python3 \
-          py3-pip \
-          make \
-          openjdk8-jre \
-          ttf-dejavu \
-          graphviz
+RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash - \
+    && yum -y install \	   
+       openjdk8-jre \
+       ttf-dejavu \
+       graphviz \
+       alsa-lib \
+       gtk3 \
+       libXScrnSaver \
+       libappindicator \
+       libnotify \
+       git \
+       wget \
+       openssl \
+       nodejs \
+       npm \
+       which \
+       gcc-c++ make \
+       epel-release \
+    && yum clean all \
+    && rm -rf /var/cache/yum
 
 # Install PlantUML
-RUN apk add --no-cache --virtual .ssl-deps \
-      openssl \
-      ca-certificates \
-    && mkdir -p /opt \
+RUN mkdir -p /opt \
     && wget -O "/opt/plantuml.jar" "https://sourceforge.net/projects/plantuml/files/plantuml.jar" \
```

Comment:
I think you could have a separate layer and do `ADD https://sourceforge.net/projects/plantuml/files/plantuml.jar /opt/` and you could avoid having to install wget to do this

Reply:
Nevermind that suggestion. curl is already in use above (`curl -sL https://rpm.nodesource.com/setup_10.x`) so using curl here instead of wget is appropriate.

That said, it might make sense to have a multi-stage Docker file (https://docs.docker.com/develop/develop-images/multistage-build/) so only the built binaries and their dependencies end up in the final image. Just a thought. For example we could try using the code in https://github.com/trustin/sphinx-binary (or a modified version?) to build the Sphinx as an executable static binary and then just copy that to the final image.

Reply:
Sounds good, will work on it. thanks.